templates/default/awsbatchcli.sh.erb

```diff
@@ -0,0 +1,8 @@
+#
+# awsbatchcli.sh:
+#   Setup AWSBatch CLI environment variables
```

Comment:
Let's always refer to this as the ParallelCluster AWSBatch CLI just because this is not an offcial AWS Batch CLI. Same applies everywhere else